docs/general/profiles.md

```diff
@@ -49,6 +50,55 @@ They may be used in any Profiles record they apply to.
 | `rust` | [Rust](https://wikipedia.org/wiki/Rust_(programming_language)) | ![Development](https://img.shields.io/badge/-development-blue) |
 | `v8js` | [V8JS](https://wikipedia.org/wiki/V8_(JavaScript_engine)) | ![Development](https://img.shields.io/badge/-development-blue) |
 
+---
+
+`profile.mapping.symbolization` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `lines_inline` | Full, most-desired level of symbolization. [1] | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
Do we need the inline flag here?
The inline information is documented to be available on the location level:
```
  // Multiple line indicates this location has inlined functions,
  // where the last entry represents the caller into which the
  // preceding entries were inlined.
  //
  // E.g., if memcpy() is inlined into printf:
  //    line[0].function_name == "memcpy"
  //    line[1].function_name == "printf"
```
And even if we want it, would it be possible to have `symbols` together with `inline` in the future or with other agent than ebpf-profiler?